src/resources/SecurityCache/SecurityCacheInterfaces.ts

```diff
@@ -0,0 +1,100 @@
+export interface SecurityCacheScheduleModel {
+    enabled?: boolean;
+    frequency?: {
+        hourMinute?: {
+            hour?: number;
+            minute?: number;
+        };
+        monthDay?: number;
+        seed?: number;
+        type?: string;
+        value?: number;
+        weekDays?: [string];
+    };
+    id?: string;
+    organizationId?: string;
+    refreshType?: string;
+    resourceId?: string;
+    scheduleType?: string;
+}
+
+interface Status {
+    numberOfEntitiesInError?: number;
+    numberOfEntitiesProcessed?: number;
+    refreshType?: string;
+    totalNumberOfEntities?: number;
+    type?: string;
+}
+
+interface Statistics {
+    numberOfEntitiesByState?: {};
+    numberOfEntitiesInError?: number;
+    numberOfPermissionModels?: number;
+    numberOfProviders?: number;
+    totalNumberOfEntities?: number;
+}
+
+export interface SecurityCacheStatus {
+    currentStatus?: Status;
+    statistics?: Statistics;
+}
+
+export interface SecurityCacheIdentityModel {
+    data: string;
+    provider: string;
+}
+
+export interface DetailedSecurityCacheMemberModel {
+    infos?: [{key?: string; value?: string}];
+    lastUpdateDate?: number;
+    lastUpdateErrorDetail?: string;
+    lastUpdateResult?: string;
+    name?: string;
+    provider?: string;
+    state?: string;
+    type?: string;
+}
+
+export interface SecurityProviderModel {
+    cascadingSecurityProviders: {};
```

Comment:
this will need a better type, it references the CascadingSecurityProvider type
![image](https://user-images.githubusercontent.com/35579930/66858521-75db2480-ef57-11e9-80cf-c4eafaf03190.png)
![image](https://user-images.githubusercontent.com/35579930/66858602-9d31f180-ef57-11e9-9367-1e8c5f31c0fd.png)

Reply:
yeah thx !! I used the model definition of the swagger but did not realised I could directly check the network call.